sky/utils/command_runner.py

```diff
@@ -228,6 +234,12 @@ def run(
             ssh_mode: The mode to use for ssh.
                 See SSHMode for more details.
             separate_stderr: Whether to separate stderr from stdout.
+            connect_timeout: timeout in seconds for the ssh connection.
+            source_bashrc: Whether to source the bashrc before running the
+                command.
+            skip_lines: The number of lines to skip from the output. This is
```

Comment:
explicitly saying that we are skipping the **first** `skip_lines` lines?

Reply:
Added. Thanks!